src/library/simplified/extension/number-of-guests.js

```diff
@@ -124,6 +124,15 @@
             b['numberOfGuests'] = b['entity.checkout.activity.travelerInfo.numberOfTickets'];
         }
     }
+    else if (utag.isLXPymt() && b.entity.checkout && b.entity.checkout.activities.length > 0) {
```

Comment:
as discussed, I'm worried why the payment page guest logic would be different than the cko page. Also, do you think we can minimize extension code? basically pre-compute as much as we can in the UDO. Don't forget we are pushing hard for server side processing, and extension coding is an anti-pattern in that regard